modules/gcloud/gcloud.go

```diff
@@ -69,6 +71,22 @@ func WithProjectID(projectID string) Option {
 	}
 }
 
+// WithDataYamlFile seeds the Bigquery project for the GCloud container.
+func WithDataYamlFile(dataYamlFile string) testcontainers.CustomizeRequestOption {
```

Comment:
suggestion: remove the duplication between function name and paramater

Reply:
Yes, I would suggest just `filename`.

However based on recent discussions and changes in other modules should we pass a reader instead?

Reply:
Yeah, makes sense. Let me change it right now. We can just simply generate a random name for the container file, can't we?

Reply:
Yep or use a fixed name, not sure what happens if there are multiple, that might be valid.